runtime/doc/if_lua.txt

```diff
@@ -246,6 +246,11 @@ Lua interfaces Vim through the "vim" module. Currently it only has `api`
 submodule which is a table with all API functions. Descriptions of these 
 functions may be found in |api.txt|.
```

Comment:
"Currently it only has `api` submodule" is  contradicted by `vim.stricmp` now, right?